src/app/components/login/login.component.css

```diff
@@ -0,0 +1,9 @@
+.loginCss {
```

Comment:
we don't use camelCase here.
it's not wrong bt we should go by the convention. i.e. login-css